.github/BE_COVENTION_CHECKLIST.md

```
@@ -0,0 +1,126 @@
+## 📋 목차
+- [패키지 구조](#패키지-구조)
+- [코드 스타일](#코드-스타일)
+- [클래스 구조](#클래스-구조)
+- [메서드 작성](#메서드-작성)
+- [테스트 코드](#테스트-코드)
+
+---
+
+## 패키지 구조
+
+### ✅ 계층형 아키텍처 준수
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)